src/lib/operators/join_hash.cpp

```diff
@@ -597,8 +604,53 @@ class JoinHash::JoinHashImpl : public AbstractJoinOperatorImpl {
         _mode(mode),
         _column_ids(column_ids),
         _predicate_condition(predicate_condition),
-        _inputs_swapped(inputs_swapped),
-        _radix_bits(radix_bits) {}
+        _inputs_swapped(inputs_swapped) {
+    /*
```

Comment:
Please fix formatting

Reply:
(below as well)

Reply:
Whoot? I ran the formatting thingy ... strange.

Reply:
That won't touch comments.